### PR TITLE
Android: Add title for the Cookie management screen

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205073613446674/f

### Description
Added title to the webview activity when displaying the cookie pop-up management web page.

### Steps to test this PR

- [x] Install from this branch.
- [x] Launch the app.
- [x] Open the Settings screen.
- [x] Tap on the "Cookie Pop-Up Protection".
- [x] Tap on the "Learn more" link. Notice the screen title is `Cookie Pop-Up Protection`.

### CODE REVIEW

**Comment by anikiki** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3350** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3350" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3350?utm_source=stack-comment).

File: autoconsent/autoconsent-impl/src/test/java/com/duckduckgo/autoconsent/impl/ui/AutoconsentSettingsViewModelTest.kt [hunk 1]
```diff
@@ -45,6 +45,15 @@ class AutoconsentSettingsViewModelTest {
         }
     }
 
+    @Test
+    fun whenOnLearnMoreSelectedCalledThenLaunchLearnMoreWebPageCommandIsSent() = runTest {
```
Comment (anikiki) [inline autoconsent/autoconsent-impl/src/test/java/com/duckduckgo/autoconsent/impl/ui/AutoconsentSettingsViewModelTest.kt:49]: Added missing test.

File: autoconsent/autoconsent-impl/src/main/java/com/duckduckgo/autoconsent/impl/ui/AutoconsentSettingsActivity.kt [hunk 2]
```diff
@@ -98,33 +98,33 @@ class AutoconsentSettingsActivity : DuckDuckGoActivity() {
...
         val options = ActivityOptions.makeSceneTransitionAnimation(this).toBundle()
-        globalActivityStarter.start(this, WebViewActivityWithUrlParam(url), options)
+        globalActivityStarter.start(this, WebViewActivityWithParams(it.url, getString(it.titleId)), options)
     }
 
     private fun configureClickableLink() {
-        val htmlGPCText = getString(R.string.autoconsentDescription).html(this)
-        val gpcSpannableString = SpannableStringBuilder(htmlGPCText)
-        val urlSpans = htmlGPCText.getSpans(0, htmlGPCText.length, URLSpan::class.java)
+        val htmlText = getString(R.string.autoconsentDescription).html(this)
```
Comment (anikiki) [inline autoconsent/autoconsent-impl/src/main/java/com/duckduckgo/autoconsent/impl/ui/AutoconsentSettingsActivity.kt:111]: Renamed variables, no other changes.

File: autoconsent/autoconsent-api/src/main/java/com/duckduckgo/autoconsent/api/WebViewScreens.kt [hunk 3]
```diff
@@ -21,6 +21,7 @@ import com.duckduckgo.navigation.api.GlobalActivityStarter
 /**
  * Use this model to launch the WebView screen
  */
-data class WebViewActivityWithUrlParam(
+data class WebViewActivityWithParams(
     val url: String,
+    val title: String,
```
Comment (anikiki) [inline autoconsent/autoconsent-api/src/main/java/com/duckduckgo/autoconsent/api/WebViewScreens.kt:26]: Added the title to the params and renamed the class to make it clear.

**Review by aitorvs — APPROVED** [review]
lgtm

**Comment by aitorvs** [comment]
LGTM!.
This will get a bit refactor when we merge https://github.com/duckduckgo/Android/pull/3274.
`autoconsent-api` should not be home for the model to launch the webViewActivity. But it's simpler if we tackle that directly in https://github.com/duckduckgo/Android/pull/3274 as it's already done in there.

**Pull request merged**